docs/core_docs/docs/integrations/toolkits/sql.ipynb

```diff
@@ -0,0 +1,305 @@
+{
+ "cells": [
+  {
+   "cell_type": "raw",
+   "id": "afaf8039",
+   "metadata": {
+    "vscode": {
+     "languageId": "raw"
+    }
+   },
+   "source": [
+    "---\n",
+    "sidebar_label: Sql Toolkit\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e49f1e0d",
+   "metadata": {},
+   "source": [
+    "# SqlToolkit\n",
+    "\n",
+    "This will help you getting started with the [SqlToolkit](/docs/concepts/#toolkits). For detailed documentation of all SqlToolkit features and configurations head to the [API reference](https://api.js.langchain.com/classes/langchain_agents_toolkits_sql.SqlToolkit.html). You can also find the documentation for the Python equivalent [here](https://python.langchain.com/docs/integrations/toolkits/sql_database/)\n",
+    "\n",
+    "## Setup\n",
+    "\n",
+    "If you want to get automated tracing from runs of individual tools, you can also set your [LangSmith](https://docs.smith.langchain.com/) API key by uncommenting below:\n",
+    "\n",
+    "```typescript\n",
+    "process.env.LANGCHAIN_TRACING_V2=\"true\"\n",
+    "process.env.LANGCHAIN_API_KEY=\"your-api-key\"\n",
+    "```\n",
+    "\n",
+    "### Installation\n",
+    "\n",
+    "This toolkit lives in the `langchain` package. You'll also need to install the `typeorm` peer dependency.\n",
```

Comment:
These should be moved to community no?

Reply:
https://python.langchain.com/v0.2/docs/integrations/toolkits/sql_database/

Reply:
Fine w/ it as a followup

Reply:
Atm all our toolkits are in langchain, py has them in community